Script/install.sh

```diff
@@ -115,6 +116,8 @@ install_dependencies() {
 	sudo apt-get install python-rpi.gpio -y
 	sudo apt-get install python3-rpi.gpio -y
 	sudo pip install -U RPi.GPIO
+  sudo pip2 install numpy
+  sudo pip3 install numpy
```

Comment:
It's the same thing here. No indentation issues. 
I find this thing a little bit worrisome.
![good_indentation](https://user-images.githubusercontent.com/26958764/27501026-aa022416-5873-11e7-97d3-193f43c1eabe.PNG)